commit ed5649f64a3f8c2e8269f9c4435e9174c4e8c775
Author: William Cohen <wcohen@redhat.com>
Date:   Thu May 9 12:23:54 2024 -0400

    Support kernels that backported kallsym functions (part 2)
    
    Git commit b8d4274d1e769780 omitted a test for the Linux 6.4 version
    of kallsyms function in runtime/linux/kprobes.c.

diff --git a/runtime/linux/kprobes.c b/runtime/linux/kprobes.c
index 2fba61cbb..9ae5565e3 100644
--- a/runtime/linux/kprobes.c
+++ b/runtime/linux/kprobes.c
@@ -855,7 +855,7 @@ stapkp_refresh(const char *modname,
          mutex_lock(&module_mutex);
 #endif
 	 kallsyms_on_each_symbol(stapkp_symbol_callback, &sd);
-#if defined(STAPCONF_KALLSYMS_6_3)
+#if defined(STAPCONF_KALLSYMS_6_3) || defined(STAPCONF_KALLSYMS_6_4)
 	 module_kallsyms_on_each_symbol(sd.modname, stapkp_symbol_callback, &sd);
 #elif LINUX_VERSION_CODE >= KERNEL_VERSION(5,12,0)
 	 module_kallsyms_on_each_symbol(stapkp_symbol_callback, &sd);
